multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/auditlogging/impl/AuditLogManager.java

```diff
@@ -53,15 +57,34 @@ Exception getException() {
     }
 
     private Logger setUpLogger(DataSource dataSource, UserInfoProvider userInfoProvider, String name) {
-        Logger logger = Logger.getLogger(name);
-        DBAppender auditLogAppender = new DBAppender(dataSource,
-                                                     AUDIT_LOG_INSERT_STATEMENT,
-                                                     EVENT_ADAPTER,
-                                                     exceptionHandler,
-                                                     userInfoProvider);
-        auditLogAppender.setName(name);
-        logger.addAppender(auditLogAppender);
-        return logger;
+        try (LoggerContext loggerContext = new LoggerContext(name)) {
+            DBAppender auditLogAppender = initializeDBAppender(dataSource, AUDIT_LOG_INSERT_STATEMENT, EVENT_ADAPTER, exceptionHandler,
+                                                               userInfoProvider, name);
+            auditLogAppender.start();
+            loggerContext.getConfiguration()
+                         .addAppender(auditLogAppender);
+            LoggerConfig loggerConfig = initializeLoggerConfig(loggerContext);
+            addAppenderToRootLogger(loggerContext, auditLogAppender);
+            return (Logger) LogManager.getLogger(loggerConfig);
```

Comment:
Did you make tests with auditlog service bound to Deploy Service application?

Reply:
Yes. The logs are stored correctly even with auditlog service.